.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -2,7 +2,7 @@
 name: Bug report
 about: Broken or unintended behavior with one of our libraries.
 title: ''
-labels: bug
+labels: question
```

Comment:
We could also add another label such as "unconfirmed bug", which gets replaced with "bug" once confirmed.

Reply:
I like this option, start with `Bug: Unconfirmed` and then manually change to `Question` and `Bug: Confirmed` or just `Bug`

Big fan of changing this regardless of what we decide to change it to!

Reply:
I am leaning towards "unconfirmed" too instead of "question". +1 for changing this.

Reply:
+1 on unconfirmed!

Reply:
```suggestion
labels: bug-unconfirmed
```